src/vector/v.to.rast.extern/v.to.rast.extern.html

```diff
@@ -0,0 +1,191 @@
+geographic region is correctly set, and that the region resolution is at 
+the desired level. Alternatively, the user can use the <b>-v</b> flag 
+to import the whole vector layer. To ensure 
+that the resulting raster layer  cleanly aligns with the computational 
+region, the extent may be slightly larger than that of the vector 
+layer.
+
+<p>
+If the coordination reference system (CRS) of the vector file differs 
+from that of the mapset in which you want to import the raster, the 
+vector file will first be reprojected using <em>ogr2ogr</em>.
+
+<p>
+By default, the object in the vector layer will be assigned a value of 
+<i>1</i> in the resulting raster layer. The user can change this value 
+using the <b>value</b> parameter. Alternatively, the user can specify 
+the name of an existing column from the vector map's attribute table. 
+In that case, the values in that column will be used as raster values 
+in the output raster map.
+
+<p>
+You can use the <b>label_column</b> parameter to assign raster category 
+labels. The user should check if each unique value from the category 
+column has one corresponding label in the label column. If there are 
+categories with more than one label, the first from the label column 
+will be used (and a warning will be printed).
+
+<p>
+With the <b>-d</b> flag, all pixels touched by lines or polygons will 
+be updated, not just those on the line render path, or whose center 
+point is within the polygon. For lines, this is similar to setting the 
+<i>-d</i> flag in <em>v.to.rast</em>.
+
+<h2>EXAMPLE</h2>
+
+The examples of <em>v.to.rast.extern</em> use vector layers from the 
+(<a href="https://grass.osgeo.org/download/data/">North Carolina sample 
+data set</a>). 
+
+<h3>Example 1</h3>
+
+First, export a vector layer as a GeoPackage. 
+
+<div class="code"><pre>
+# Export the geology vector layer as Geopackage
+v.out.ogr input=geology@PERMANENT output=geology.gpkg format=GPKG
+</pre></div>
+
+<p>
+Import the geology.gpkg as raster. Raster cells overlapping with the 
+vector features will be assigned a value of 1, and the other raster 
+cells Nodata. If you have RAM to spare, increase the memory to speed up 
+the import.
+
+<div class="code"><pre>
+# Set the region
+g.region -a vector=geology res=500
+
+# Import the layer
+v.to.rast.extern input=geology.gpkg \
+output=geology_rast \
+memory=20000
+</pre></div>
+
+<div align="left" style="margin: 10px"> <a 
+href="v_to_rast_extern_im01.png"> <img src="v_to_rast_extern_im01.png" 
+alt="The geology vector file converted to, and imported as raster in 
+GRASS. Example 1" border="0"> </a><br><i>Figure 1: The geology vector file was 
+converted to, and imported as a raster in GRASS, using the default 
+settings.</i> </div>
+
+<p>
+Note that if you have a GeoPackage (or any other data source) with 
+multiple layers, you need to specify which layer you want to use with 
+the <b>layer</b> parameter. If you do not, the first layer will be 
+selected. 
```

Comment:
```suggestion
If the GeoPackage file (or any other data source) has  
multiple layers, users need to specify which layer to use with 
the <b>layer</b> parameter. Otherwise, the first layer will be 
selected. 
```